cfripper/rules/rds_security_group.py

```diff
@@ -0,0 +1,93 @@
+from typing import Optional, Union
+
+from pycfmodel.model.resources.security_group import RDSDBSecurityGroup
+from pycfmodel.model.resources.security_group_ingress import RDSDBSecurityGroupIngress
+
+from cfripper.model.enums import RuleGranularity, RuleMode, RuleRisk
+from cfripper.model.result import Result
+from cfripper.rules import ResourceSpecificRule
+
+
+class RDSSecurityGroupIngressOpenToWorldRule(ResourceSpecificRule):
+    """
+    Checks if security groups have an ingress IP that is open to the world an RDS security group.
```

Comment:
This sentence looks a little bit confusing to me.
```suggestion
    Checks if RDS native security groups have an ingress IP that is open to the world.
```